src/super_gradients/common/environment/env_variables.py

```diff
@@ -45,5 +46,9 @@ def HYDRA_FULL_ERROR(self) -> Optional[str]:
     def HYDRA_FULL_ERROR(self, value: str):
         os.environ["HYDRA_FULL_ERROR"] = value
 
+    @property
+    def SUPER_GRADIENTS_LOG_DIR(self) -> str:
+        return os.getenv("SUPER_GRADIENTS_LOG_DIR", default=f"{Path.home()}/sg_logs")
```

Comment:
Oops i missed that, the `/` is part of the strings. This is not well handled in some os.
Instead you can either `str(Path.home()/"sg_logs")` or `os.path.join(Path.home(), "sg_logs")`

Reply:
Done 👍
I chose the first option since I prefer pathlib 😄